content/en/docs/languages/go/generated-code.md

```diff
@@ -3,6 +3,7 @@ title: Generated-code reference
 linkTitle: Generated code
 weight: 95
 ---
+The following documents the latest generated code guide which uses generics.  For documentation about the older version of generated code that did not use generics, please see the doc for [old generated code](/docs/languages/go/generated-code-old).
 
 This page describes the code generated with the [grpc plugin](https://pkg.go.dev/google.golang.org/grpc/cmd/protoc-gen-go-grpc), `protoc-gen-go-grpc`,
```

Comment:
Let's reverse the order of these two paragraphs, combine them into one, and make them flow better.

Reply:
Done